common/rest/request.go

```diff
@@ -51,12 +52,14 @@ import (
 	"net/textproto"
 	"net/url"
 	"strings"
+
+	"github.com/IBM-Cloud/ibm-cloud-cli-sdk/bluemix/models"
 )
 
 const (
-	contentType               = "Content-Type"
-	jsonContentType           = "application/json"
-	formUrlEncodedContentType = "application/x-www-form-urlencoded"
+	ContentType               = "Content-Type"
+	JSONContentType           = "application/json"
+	FormUrlEncodedContentType = "application/x-www-form-urlencoded"
 )
```

Comment:
**Callout**:  Need to export variables to allow using models package in request.go